### PR TITLE
impl Display for ClientId

### PR DESCRIPTION
**Pull request by Luminoth**
Pretty simple, just impl's Display for ClientId.

### CODE REVIEW

File: src/core/connected_clients.rs [hunk 1]
```diff
@@ -12,18 +12,18 @@ pub struct ConnectedClients(Vec<ClientId>);
 
 impl ConnectedClients {
     pub(crate) fn add(&mut self, client_id: ClientId) {
-        debug!("adding connected `{client_id:?}`");
+        debug!("adding connected `{client_id}`");
```
Comment (UkoeHB) [inline src/core/connected_clients.rs:15]: This changes from `"adding connected ClientId(10)"` to `"adding connected 10"`, which is not what we want.

Comment (Luminoth) [reply src/core/connected_clients.rs:15]: Sure, I wasn't super confident that change was wanted in the output.

**Comment by Shatur** [comment]
@UkoeHB what do you think about the suggestion in general? 
Looks like renet implements `Display` for it, but I'm not sure if it worth adding something like this 🤔

**Comment by UkoeHB** [comment]
From the [rust docs](https://doc.rust-lang.org/std/fmt/trait.Display.html): "Display is similar to Debug, but Display is for user-facing output, and so cannot be derived."

It's hard for me to imagine a case of user-facing output that would benefit from a Display implementation here. The struct is simple enough that you can just deref it to access the inner integer, if that's all you want to display.

**Comment by Shatur** [comment]
Agree!

**Pull request closed without merging**